tests/integration/components/paper-dialog-test.js

```diff
@@ -35,7 +35,7 @@ module('Integration | Component | paper dialog', function(hooks) {
       {{/paper-dialog}}
     `);
 
-    let dialogContent = $('md-dialog').html().trim();
+    let dialogContent = find('md-dialog').innerHTML.trim();
```

Comment:
With `qunit-dom` (already a dependency) you can write `assert.dom('md-dialog').asText('Lorem ipsum.')`